explorer/app/components/Index/common/indexTransformer.ts

```diff
@@ -27,11 +27,8 @@ export function getPluralizedMetadataLabel(
  */
 export function getSummaries(
   summaries: Array<keyof typeof SUMMARY>,
-  summaryResponse?: SummaryResponse
-): Summary[] | undefined {
-  if (!summaryResponse) {
-    return;
-  }
+  summaryResponse: SummaryResponse
```

Comment:
`summaryResponse` is guaranteed to be specified due to the guard in [views/Index/index.tsx](https://github.com/clevercanary/data-browser/pull/210/files#diff-0aba8d015292bd218ec8032a938ac644f0146e6d666f1fdec0734ced0c5b8517L31).